index.js

```diff
@@ -207,16 +214,16 @@ client.on('message', async message => {
     }

```

Comment:
remove this if statement block since line 217 now checks for it

Reply:
Actually, it's better to move the check up here as a separate check, ie if(notMusic.includes(cmd)) { return cmd call message; }

that way we can keep the proper error message for the user in the next if statement